src/calliope/preprocess/load.py

```diff
@@ -42,12 +34,13 @@ def load_model_definition(
             If not None, dictionary of overrides to apply.
             These will be applied _after_ `scenario` overrides.
             Defaults to None.
+        **kwargs: initialisation overrides.
 
     Returns:
         tuple[AttrDict, Optional[Path], str]:
-            1. Model definition with overrides applied.
-            1. Path to model definition YAML if input `model_definiton` was pathlike, otherwise None.
-            1. Expansion of scenarios (which are references to model overrides) into a list of named override(s) that have been applied.
+            - Model definition with overrides applied.
```

Comment:
I previously had it numbered so that it would be obvious the order of items in the tuple, is bullet points recommended instead?

Reply:
Oh, I'll re introduce them. They all had 1 so I thought it was fine.